xwiki-platform-core/xwiki-platform-mail/xwiki-platform-mail-send/xwiki-platform-mail-send-default/src/test/java/org/xwiki/mail/script/ScriptMimeMessageTest.java

```diff
@@ -90,4 +91,14 @@ public void addHeader() throws Exception
 
         assertEquals(testHeader, this.scriptMessage.getHeader("X-Test", null));
     }
+    
+    @Test
+    public void addPart() throws Exception
+    {
+        String header = "bodyPartTest";
+        MimeBodyPart bp = new MimeBodyPart();
+        bp.addHeader("BodyPart-Test", header);
+        this.scriptMessage.addPart(bp);
+        assertEquals(header, this.scriptMessage.getHeader("BodyPart-Test", null));
+    }
```

Comment:
You should also add a test for the other method. I know right now this method use the other one but it might change and we need to make sure if someone refactor something the method keep supporting BodyPart.

Reply:
Done!